docs/Guides/v1_v2_migration.md

```diff
@@ -20,3 +20,22 @@ production, you should:
 1. Duplicate the `basistheory_reactor` with the same `code` as the corresponding `basistheory_reactor_formula`
 2. Change your production systems to start using the newly created `basistheory_reactor`
 3. Delete the old reactor
+
+## basistheory_application `create_key` and basistheory_application_keys
+
+In `v2` we have introduced the concept of [Application Keys](https://developers.basistheory.com/docs/api/applications/application-keys), which allow multiple keys to be created for an application - enabling 
+you to rotate keys without downtime. With this addition, we've added a `create_key` property to the `basistheory_application` resource 
+which is only used when creating a new application. 
+
+- When creating a new `basistheoyr_application` in v2
+  - We strongly suggest you leave the `create_key` default
+  - Create a `basistheory_application_key` along with this, to get full lifecycle management of the key. 
+- If you are migrating from a `v1` basistheory_application to `v2`
+  - You will need to update your `basistheory_application` with the `create_key` property set to `true`
+  - You are able to also create a `basistheory_application_key` for this application
+  - ⚠️ Keep in mind that these applications will have an Unmanaged Key, which means you will need to manage the key lifecycle yourself in the portal.
```

Comment:
Using a bullet point structure to organize this information makes it a bit confusing. Maybe use level 3 headings for the branching "if"?

<img width="1107" alt="Screenshot 2024-04-19 at 10 16 01" src="https://github.com/Basis-Theory/terraform-provider-basistheory/assets/5549408/7c989947-c9fa-4e6a-a5b4-86d9abfee039">

Reply:
Bullet points or numbered lists could be used for steps.